tls/extensions/s2n_sever_signature_algorithms.c

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
This file is called "sever" instead of "server"

Reply:
spelling is hard haha